ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/TransactionPoolTest.java

```diff
@@ -796,10 +807,12 @@ public void shouldIgnoreEIP1559TransactionBeforeTheFork() {
     when(protocolSpec.getFeeMarket()).thenReturn(FeeMarket.legacy());
     when(transactionValidator.validate(any(Transaction.class), any(Optional.class), any()))
         .thenReturn(valid());
-    when(transactionValidator.validateForSender(
-            any(Transaction.class),
-            nullable(Account.class),
-            any(TransactionValidationParams.class)))
+    lenient()
```

Comment:
shouldIgnoreEIP1559TransactionBeforeTheFork  validateForSender() doesn't get called because the tx is rejected before that